code/controllers/subsystem/init/more_init_stuff.dm

```diff
@@ -20,10 +20,15 @@ var/datum/subsystem/more_init/SSmore_init
 	create_global_parallax_icons()
 	log_startup_progress("  Finished caching space parallax simulation in [stop_watch(watch)]s.")
 
-	watch=start_watch()
-	log_startup_progress("Generating holominimaps...")
-	generateHoloMinimaps()
-	log_startup_progress("  Finished holominimaps in [stop_watch(watch)]s.")
+	if (!config.skip_minimap_generation)
+		watch=start_watch()
+		log_startup_progress("Generating holominimaps...")
+		generateHoloMinimaps()
+		log_startup_progress("  Finished holominimaps in [stop_watch(watch)]s.")
+	else
+		holomaps_initialized = 1 //Assume holominimaps were prerendered, the worst thing that happens if they're missing is that the minimap consoles don't show a minimap
+		log_startup_progress("Not generating holominimaps - SKIP_HOLOMINIMAP_GENERATION found in config/config.txt")
+	..()
```

Comment:
why

Reply:
I'm sure having two supercalls in there breaks something, but I'm curious what

Reply:
```
/datum/subsystem/proc/Initialize(start_timeofday)
	var/time = (world.timeofday - start_timeofday) / 10
	var/msg = "Initialized [name] subsystem within [time] seconds!"
	log_debug(msg,FALSE) // Redundant chat output as whole world sees message below anyways
	world.log << msg
	to_chat(world, "<span class='danger'>[msg]</span>")
	initialized = TRUE
	return time
```
It's fine.

Reply:
Well the message is duplicated, but that's not really a big deal

Reply:
I thought that was the intent.

Reply:
But for what purpose?

Reply:
I was under the impression that it was a typo